libraries/libfc/src/crypto/bls_signature.cpp

```diff
@@ -17,7 +17,7 @@ namespace fc::crypto::blslib {
 
          std::array<uint8_t, 192> bytes = fc::crypto::blslib::deserialize_base64<std::array<uint8_t, 192>>(data_str);
 
-         constexpr bool check = false; // default
+         constexpr bool check = true; // check if base64str is invalid
          constexpr bool raw = true;
          std::optional<bls12_381::g2> g2 = bls12_381::g2::fromAffineBytesLE(bytes, check, raw);

```

Comment:
Not related to this PR, but I don't think the APIs from bls12_381 are well designed. These bool parameters could be replaced by enum template parameters which would be clearer and more efficient.